BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01609-03
\ No newline at end of file
+2.0.0-prerelease-01616-05
```

Comment:
Why did you pick this version? We should align to what is in source build (https://github.com/dotnet/source-build/blob/dev/release/2.0/BuildToolsVersion.txt) which is 2.0.0-prerelease-01721-01.

Reply:
I got it from https://github.com/dotnet/source-build/blob/dev/release/2.0/patches/standard/0002-Unify-CLI-and-BuildTools-Versions.patch

Reply:
I think we need to align with source-build so that when we start sharing the same versions things work as expected.

Reply:
Do you have any ideas on how we keep this toolset coherent between all the repos and source-build?  Manually keeping them together isn't working today.

Note: This is a larger issue than just source-build, but source-build is driving the importance of this issue.

Reply:
That is definitely going to be a challenge but I think we can probably enforce it at the source-build CI level by doing some validation to ensure they all match and if they don't we don't take the sha update.